meson.build

```diff
@@ -185,3 +380,12 @@ pgroonga_database_update_sqls = files(
 install_data(pgroonga_database_update_sqls,
   install_dir: pg_sharedir / 'extension',
 )
+
+pgroonga_scripts = files(
+  'tools/pgroonga-primary-maintainer.sh',
+  'tools/systemd/pgroonga-generate-primary-maintainer-service.sh',
+  'tools/systemd/pgroonga-generate-primary-maintainer-timer.sh',
+)
+install_data(pgroonga_scripts,
+  install_dir: pg_bindir,
+)
```

Comment:
Should we add this in this PR?

Reply:
fix: [be1e768](https://github.com/pgroonga/pgroonga/pull/750/commits/be1e768407d147cdc83e27fda911f89eb0919b93) We don't need them now. So I removed them.